_data/melwinlobo18/melwinlobo18.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Melwin Lobo
+institution: Sahyadri College of Engineering and Management
+profile_pic: melwin.jpg
+quote: “We do not need magic to change the world, we carry all the power we need inside ourselves already”
```

Comment:
Please follow the given instruction by removing the quotes to maintain the same format